Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/azure/serverless/spark/models/ApiVersion.java

```diff
@@ -24,4 +24,5 @@
 
 public class ApiVersion {
     public static final String VERSION = "2018-02-01-preview"; // Preview version
+    public static final String JOB_API_VERSION = "2016-11-01";
```

Comment:
There is a special ApiVersion in job api, otherwise, create one for it.

Reply:
Double confirmed, there is no `ApiVersion` class in Job API though I find one in account API.

---
In reply to: [200133979](https://github.com/Microsoft/azure-tools-for-java/pull/1756#discussion_r200133979) [](ancestors = 200133979)